index.js

```diff
@@ -54,8 +69,18 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
-
+function getAverageIMDBRating(movies) {
+  allAverages = 0
+  for (let i = 0; i < movies.length; i++ ){
+    if ( movies[i].imdbRating ){
+  allAverages += parseFloat(movies[i].imdbRating) ;
+  }
+  else {
+    return 0
```

Comment:
This here could cause some issues and end our loop is `movies[i].imdbRating` was ever null, 0, or some other falsy value.